amit/html/Assigment3/Amit3_3.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang ="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta hhtp-equiv= "X-UA-Compatible" content="IE=edge">
+    <meta name="viewport"  content= "width=device-width ,initial-scale=1.0">
```

Comment:
remove extra space present before content and remove space before , make this change in all files